### ISSUE TITLE
Cannot import after installation. Please help me

### ISSUE THREAD

**Issue by RedRx** [devblackops/Terminal-Icons#89]
![image](https://user-images.githubusercontent.com/73677545/201457042-7b5ef2e2-8d50-42ce-8889-072f9b150858.png)
![image](https://user-images.githubusercontent.com/73677545/201457030-2be3e60d-7488-4df3-84eb-4e472dd57a83.png)

Thank you very much

**Comment by devblackops** [comment]
@RedRx Please try the solution in https://github.com/devblackops/Terminal-Icons/issues/77.

**Comment by NeiL978** [comment]
try this in your profile.ps1
```
# Terminal-icons
Import-Module Terminal-Icons
```

**Comment by Arthur-Carbonari** [comment]
instead of using Import-Module Terminal-Icons try Import-Module [absolute path]/Terminal-Icons.psm1

**Comment by JuanMaChico** [comment]
![Untitled](https://user-images.githubusercontent.com/49964403/210852330-28ed6d5a-f805-4cc1-bdd2-8262f9f56d59.jpg)

Me muestra asi y no me muestra los Iconos, y mi implementacion esta asi

oh-my-posh init pwsh --config 'C:\Users\Juan Manuel\AppData\Local\Programs\oh-my-posh\themes\clean-detailed.omp.json' | Invoke-Expression
Import-Module -Name Terminal-Icons

**Comment by Arthur-Carbonari** [comment]
> ![Untitled](https://user-images.githubusercontent.com/49964403/210852330-28ed6d5a-f805-4cc1-bdd2-8262f9f56d59.jpg)
> 
> Me muestra asi y no me muestra los Iconos, y mi implementacion esta asi
> 
> oh-my-posh init pwsh --config 'C:\Users\Juan Manuel\AppData\Local\Programs\oh-my-posh\themes\clean-detailed.omp.json' | Invoke-Expression Import-Module -Name Terminal-Icons

You need to have a "nerd" font installed and set as the terminal font.

**Comment by JuanMaChico** [comment]
> 

La letra es  Hack Nerd Font Mono
![image](https://user-images.githubusercontent.com/49964403/211568363-da2a6094-0381-4722-817d-2a0c420a9dee.png)

**Comment by JuanMaChico** [comment]
![image](https://user-images.githubusercontent.com/49964403/211569203-679b1e09-617a-4eca-8115-b50509418ce7.png)

**Comment by RedRx** [comment]
Now It's success. I'm think it's not support on Windows 11.

I'm try to install Windows 10 and install Terminal Icons is work !.

After that I upgrade Win10 to 11 and everything is working.

Thank you

**Comment by hkrobotics** [comment]
> Now It's success. I'm think it's not support on Windows 11.
> 
> I'm try to install Windows 10 and install Terminal Icons is work !.
> 
> After that I upgrade Win10 to 11 and everything is working.
> 
> Thank you

Is there a short fix for this issue instead of going through this process of hobbing back and forth between W11 and W10?

**Comment by frivard-coveo** [comment]
I am on Windows 11 and it works quite well.
Your error is most probably due to `oh-my-posh init pwsh` in your profile. That's not really how modules should be used. You should _install_ them once, from inside a regular session, then _load_ them inside your profile file. For example, here my profile file:
```
Import-Module oh-my-posh
Import-Module -Name Terminal-Icons
Set-PoshPrompt -Theme "~\Documents\Powershell\myscustomtheme.omp.json"
```

See? No install, no init pwsh. Just Import an existing module.

**Comment by atakiya** [comment]
> I am on Windows 11 and it works quite well. Your error is most probably due to `oh-my-posh init pwsh` in your profile. That's not really how modules should be used. You should _install_ them once, from inside a regular session, then _load_ them inside your profile file. For example, here my profile file:
> 
> ```
> Import-Module oh-my-posh
> Import-Module -Name Terminal-Icons
> Set-PoshPrompt -Theme "~\Documents\Powershell\myscustomtheme.omp.json"
> ```
> 
> See? No install, no init pwsh. Just Import an existing module.

@frivard-coveo Except that is exactly what oh-my-posh recommends:
https://ohmyposh.dev/docs/installation/prompt
**Unless you are using the module, which is *deprecated*** (among just being a wrapper for the exe for some time now)
![image](https://user-images.githubusercontent.com/6952402/226074587-9a1cf497-4756-4f6d-85f9-6d5702ec2554.png)

<details><summary>(expand me)<br/>And works fine on Windows 11 + Powershell v7 + oh-my-posh + Terminal-Icons</summary>

![image](https://user-images.githubusercontent.com/6952402/226074801-5bf8a46a-eeb9-434f-8023-c7cae9c83a65.png)

</details>

So this has to have some other cause with this certain configuration, not `oh-my-posh`, nor Win11.

**Comment by no5sunshine1029** [comment]
> ![Untitled](https://user-images.githubusercontent.com/49964403/210852330-28ed6d5a-f805-4cc1-bdd2-8262f9f56d59.jpg)
> 
> Me muestra asi y no me muestra los Iconos, y mi implementacion esta asi
> 
> oh-my-posh init pwsh --config 'C:\Users\Juan Manuel\AppData\Local\Programs\oh-my-posh\themes\clean-detailed.omp.json' | Invoke-Expression Import-Module -Name Terminal-Icons

I have the same problem and solved it, you can try my solution[#99](https://github.com/devblackops/Terminal-Icons/issues/99)
